flink-shaded-curator/src/main/resources/META-INF/NOTICE

```diff
@@ -6,7 +6,7 @@ The Apache Software Foundation (http://www.apache.org/).
 
 This project bundles the following dependencies under the Apache Software License 2.0 (http://www.apache.org/licenses/LICENSE-2.0.txt)
 
-- com.google.guava:guava:16.0.1
-- org.apache.curator:curator-client:2.12.0
-- org.apache.curator:curator-framework:2.12.0
-- org.apache.curator:curator-recipes:2.12.0
+- com.google.guava:guava:27.0.1-jre
```

Comment:
Unexpected modification?

Reply:
> Unexpected modification?

yes, curator-4.2.0 depend on guava-27.0.1, and flink-shaded-curator will relocate `com.google.common` to `org.apache.flink.curator.shaded.com.google.common`, so it has no effect.

Reply:
A better way would be excluding guava that curator depends on. I don't think we need this transitive dependency.

Reply:
> A better way would be excluding guava that curator depends on. I don't think we need this transitive dependency.

I am not sure that wheather curator-4.2.0 use some special guava's api or not, I am glad to test it offline.